scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,100 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
+from openstates.models import ScrapeCommittee
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        try:
+            for members_area in XPath(
+                '//*[contains(@class, "committeeMemberList")]'
+            ).match_one(self.root)[0]:
+                if "There are no members" in members_area.text:
+                    break
```

Comment:
`break` or `continue` here? `break` obviously means any lines _after_ the one you found will be missed.

Reply:
yes break... no need to look into other divs if there is text mentioning that there are no members